sdcm/utils/common.py

```diff
@@ -1949,3 +1950,13 @@ def ec2_instance_wait_public_ip(instance):
     if instance.public_ip_address is None:
         raise PublicIpNotReady(instance)
     LOGGER.debug(f"[{instance}] Got public ip: {instance.public_ip_address}")
+
+
+def ec2_ami_get_root_device_name(image_id, region):
+    ec2 = boto3.resource('ec2', region)
+    image = ec2.Image(image_id)
+    try:
+        if image.root_device_name:
+            return image.root_device_name
+    except (TypeError, ClientError):
+        raise AssertionError(f"Image '{image_id}' details not found in '{region}'")
```

Comment:
Exceptions will be chained. Do you expect this behavior? Otherwise, I can suggest to use `raise ... from None` syntax.

Reply:
Yep, better to see why the Assertion happened